backend/Dockerfile

```diff
@@ -1,15 +1,14 @@
-FROM python:3.11.1-alpine3.16
+FROM python:3.11-alpine3.16
```

Comment:
I don't believe the `3.11.1-alpine3.16` tag is listed in https://hub.docker.com/_/python for the `python` image anymore (it'll still work though, I believe). However, changed the tag here to be `3.11-alpine3.16` as that would be more general and refer to the same image as the following tags: 

[`3.11.2-alpine3.16, 3.11-alpine3.16, 3-alpine3.16, alpine3.16`](https://github.com/docker-library/python/blob/e664abbb9af1ee6aae5778d70a5df79d86d9ec57/3.11/alpine3.16/Dockerfile)

Unless there's a specific reason to use the minor version of 3.11.1 for python, could use the more general tag as that'll likely be updated with newer minor versions by the image maintainers. Hell, if the minor versions don't matter much, could even use the simple `alpine` tag instead. Just a thought if it makes sense here.

Reply:
I was just trying to get a generally up to date Python image, so sounds good :)